test/test_pty.rs

```diff
@@ -0,0 +1,36 @@
+use nix::pty::openpty;
+use nix::unistd::{read, write, close};
+
+#[test]
+fn test_openpty() {
+    // TODO: figure out the right termios settings to pass in here
```

Comment:
`Default` should always set the fields to valid values, it kind of defeats the point if it doesn't. For the termios struct I'm pretty certain for the `termios` struct that all zeros is valid.

But for your tests, just create a termios struct out of uninitialized memory and use the various methods available to set the struct to your desired values. Does that work?